docs/zh_cn/reference/command_reference.md

```diff
@@ -733,16 +738,86 @@ juicefs gateway redis://localhost localhost:9000
 
 |项 | 说明|
 |-|-|
-| `--log value`<VersionAdd>1.2</VersionAdd>      | 网关日志路径                                                                                   |
+| `--log value`<VersionAdd>1.2</VersionAdd>      | 网关日志路径 |
 |`META-URL`|用于元数据存储的数据库 URL，详情查看[「JuiceFS 支持的元数据引擎」](../reference/how_to_set_up_metadata_engine.md)。|
-| `--background, -d`<VersionAdd>1.2</VersionAdd> | 后台运行 (默认：false)                                                                    |
 |`ADDRESS`|S3 网关地址和监听的端口，例如：`localhost:9000`|
 |`--access-log=path`|访问日志的路径|
-|`--no-banner`|禁用 MinIO 的启动信息 (默认：false)|
+| `--background, -d`<VersionAdd>1.2</VersionAdd> | 后台运行 (默认：false) |
+|`--no-banner`| 禁用 MinIO 的启动信息 (默认：false) |
 |`--multi-buckets`|使用第一级目录作为存储桶 (默认：false)|
 |`--keep-etag`|保留对象上传时的 ETag (默认：false)|
 |`--umask=022`|新文件和新目录的 umask 的八进制格式 (默认值：022)|
+|`--object-tag` <VersionAdd>1.2</VersionAdd> | 启用对象标签 API |
 | `--domain value`<VersionAdd>1.2</VersionAdd>   |虚拟主机样式请求的域|
+|`--refresh-iam-interval` <VersionAdd>1.2</VersionAdd>| 重新加载网关 IAM 配置的间隔时间（默认值：5 分钟）|
+
+#### 元数据相关选项 {#gateway-metadata-options}
```

Comment:
Because most of the options of the `gateway` and `webdav` commands are the same as the `mount` command, there is no need to mention it repeatedly in the document.

Reply:
After communicating with @yuhr123, it was decided to put these common options into a new Markdown file and put these common options into the "Command Reference" document through the [Markdown import feature](https://docusaurus.io/docs/markdown-features/react#importing-markdown) of Docusaurus.